src/c_aci_testing/tools/aci_deploy.py

```diff
@@ -31,13 +32,21 @@ def aci_deploy(
         add=False,
     )
 
+    bicep_file_path = None
+    bicepparam_file_path = None
+
     # Find the bicep files
     for file in os.listdir(target_path):
         if file.endswith(".bicep"):
             bicep_file_path = os.path.join(target_path, file)
         elif file.endswith(".bicepparam"):
             bicepparam_file_path = os.path.join(target_path, file)
 
+    if not bicep_file_path:
+        raise FileNotFoundError(f"No bicep file found in {target_path}")
+    if not bicepparam_file_path:
+        raise FileNotFoundError(f"No bicepparam file found in {target_path}")
```

Comment:
In normal bicep deployment, I remember sometimes you don't have to provide bicepparam file when default values are defined in the bicep file. Should we really always throw here?

Reply:
Hmm that's a good point. Although in this case if there's no bicepparam_file_path the az_command below will be wrong I think (i.e. it will have "--parameters _(empty string)_")

Reply:
can fix in next commit

Reply:
Ah, OK if the script except it then I think it can throw here

Reply:
Actually I'm not going to add more complexity to allow not having a bicepparam file, since this is testing confidential ACI and we will pretty much always have a policy, and `policies gen` writes the policy to the bicepparam